### ISSUE TITLE
Learn about the TP-MASK regions for WFPC SVM issue

### ISSUE THREAD

**Issue by stscijgbot-hstdp** [spacetelescope/drizzlepac#1656]
_Issue [HLA-1106](https://jira.stsci.edu/browse/HLA-1106) was created on JIRA by [Steve Goldman](https://jira.stsci.edu/secure/ViewProfile.jspa?name=sgoldman):_

As discussed in [HLA-625](https://jira.stsci.edu/browse/HLA-625), we need to be able to repeatably identify regions of the SVM cosmic ray issue before we can implement a fix. 

For this ticket a new method may need to be created to identify regions of no overlap for WFPC2 data, and add it to a (TBD) mask.

**Comment by stscijgbot-hstdp** [comment]
_Comment by [Steve Goldman](https://jira.stsci.edu/secure/ViewProfile.jspa?name=sgoldman) on [JIRA](https://jira.stsci.edu/browse/HLA-1106?focusedCommentId=659704#comment-659704):_

Depending on results of another ticket (HLA-1061) the above issue may not need any changes. 

In the meantime, I will use the opportunity to learn about this part of the code.
